### ISSUE TITLE
Can't install using Carthage

### ISSUE THREAD

**Issue by bpolania** [Boilertalk/secp256k1.swift#1]
I'm adding the following line to the `Carthage` file:

`github "Boilertalk/secp256k1.swift"`

and running `carthage update` and getting the following error:

```
*** Fetching secp256k1.swift
*** Checking out secp256k1.swift at "0.1.1"
*** xcodebuild output can be found in /var/folders/jx/x6pq8zvs2xqghx_whv3cyh9w7qmttx/T/carthage-xcodebuild.y4fL6H.log
*** Skipped building secp256k1.swift due to the error:
Dependency "secp256k1.swift" has no shared framework schemes

If you believe this to be an error, please file an issue with the maintainers at https://github.com/Boilertalk/secp256k1.swift/issues/new

```

**Comment by Ybrin** [comment]
I forgot to share the correct scheme. I just released version [0.1.2](https://github.com/Boilertalk/secp256k1.swift/releases/tag/0.1.2) which resolves this issue. Please try and run `carthage update` again.

**Comment by bpolania** [comment]
Tried and worked without errors.
 Thanks!
